From: Jeremy Bicha <jbicha@debian.org>
Date: Tue, 16 Oct 2018 22:55:32 -0400
Subject: build without gconf

dict/doc/Readme.mac recommends removing the AM_GCONF_SOURCE_2 lines

which also makes autoreconf happy
---
 dict/configure.ac | 2 --
 1 file changed, 2 deletions(-)

diff --git a/dict/configure.ac b/dict/configure.ac
index 3225c6e..fc7101a 100644
--- a/dict/configure.ac
+++ b/dict/configure.ac
@@ -318,7 +318,6 @@ elif test "x${enable_maemo_support}" = "xyes" ; then
 	if test x"$GCONFTOOL" = xno; then
         	AC_MSG_ERROR([gconftool-2 executable not found in your path - should be installed with GConf])
 	fi
-	AM_GCONF_SOURCE_2
 elif test "x${enable_darwin_support}" = "xyes" ; then
         echo "Enable darwin suppport";
 	AC_DEFINE([CONFIG_DARWIN],[], [Compile with darwin support])
@@ -339,7 +338,6 @@ dnl *********************************************
 	if test x"$GCONFTOOL" = xno; then
         	AC_MSG_ERROR([gconftool-2 executable not found in your path - should be installed with GConf])
 	fi
-	AM_GCONF_SOURCE_2
 
 dnl ==========================================================================
 dnl stuff for the CORBA interface

